cfgov/cfgov/urls.py

```diff
@@ -201,7 +201,8 @@
 
     url(r'^credit-cards/knowbeforeyouowe/$', TemplateView.as_view(template_name='knowbeforeyouowe/creditcards/tool.html'), name='cckbyo'),
     # Form crsf token provider for JS form submission
-    url(r'^token-provider/', token_provider)
+    url(r'^token-provider/', token_provider),
```

Comment:
Rethinking this, I decided I would make the CSP reporting conditional-- if the csp middleware isn't installed, then the URL isn't enabled.